### DISCUSSION TITLE
Indexing request rejected by Google Search Console

### DISCUSSION THREAD

**Discussion by nebkoguide** [vercel/community#2612] [Help]
### Summary

Every time I try to submit indexing request to Google Search Console, it is giving me this error 👇. Please help me. This is soo frustrating. I'm trying since a week and is giving me same problem.

![image](https://github.com/vercel/community/assets/108616642/16597940-b9d6-489c-9d50-0642d51e6b29)

![image](https://user-images.githubusercontent.com/108616642/240800128-d03fabc4-6556-42e6-8980-b4c16daf8e94.png)

I asked this question [here](https://github.com/vercel/next.js/discussions/50316) and I was advised to post it here. Please help

### Steps to Reproduce

This is one of the webpage https://nebkoguide.com/questions/what-is-the-controlling-idea-or-thesis-of-this-essay-sharing-traditio

**Comment by amyegan** [comment]
Thanks for reporting this, @nebkoguide. We posted an incident on our status site where you can find the latest updates: https://www.vercel-status.com/incidents/8w42qmnv83cj

**Reply by nebkoguide** [reply]
> Thanks for reporting this, @nebkoguide. We posted an incident on our status site where you can find the latest updates: https://www.vercel-status.com/incidents/8w42qmnv83cj

@amyegan @rauchg please help. The issue was solved, but it started again.

**Comment by nebkoguide** [comment]
@amyegan It was solved for a day, but it started showing same errors again :(

this is one of the webpage: https://nebkoguide.com/questions/it-is-not-easy-to-motivate-others-how-do-you-think-steve-jobs-speec

**Reply by damonphilipross** [reply]
I'm having the same issue as you

**Comment by gurkerl83** [comment]
Same here!
Update: After removing and re-adding the respective domain from / to the project in the vercel dashboard, it starts working again.
